example/lib/main.dart

```diff
@@ -11,6 +11,7 @@ import 'package:single_factor_auth_flutter/single_factor_auth_flutter.dart';
 import './utils.dart';
 
 void main() {
+  WidgetsFlutterBinding.ensureInitialized();
```

Comment:
Why are we doing this? I don't think it should be required.

Reply:
some concurrency issue. So I added that. Removed.